src/lib/tagHelper.ts

```diff
@@ -0,0 +1,20 @@
+const hasEventBubbledThroughTag = ({
+  tagName,
+  target,
+  currentTarget,
+}: {
+  tagName: string
+  target: HTMLElement
+  currentTarget: HTMLElement
+}) => {
+  if (target == currentTarget || !target.parentElement) return false
+  if (target.tagName.toLowerCase() === tagName.toLowerCase()) return true
```

Comment:
any chance target.tagName could be undefined?

Reply:
Even though I don't think that's possible because that would be an invalid html tag, I added this check to make sure: https://github.com/carforyou/carforyou-components-pkg/pull/373/files#diff-6023195695dc84b77588cf36adfcd004d5cc40ce91e778fbb233e0f84bb94ed7R60